lib/ui/painting/paint.cc

```diff
@@ -116,7 +122,19 @@ Paint::Paint(Dart_Handle paint_objects, Dart_Handle paint_data) {
   if (filter_quality)
     paint_.setFilterQuality(static_cast<SkFilterQuality>(filter_quality));
 
-  if (uint_data[kColorFilterIndex]) {
+  if (uint_data[kColorFilterIndex] && uint_data[kInvertColorIndex]) {
```

Comment:
I wonder if we have unit tests or golden tests in flutter/flutter that exercise the two new branches that you added?

Reply:
https://philwrightdev.wordpress.com/2014/10/01/making-clang-format-ignore-sections/

Reply:
I think the existing set of golden tests should cover everything, and then when I map the invert color flag to the accessibility setting after rolling the engine I can add special test cases? I'm not sure of another way I could write the tests for this